packages/template-retail-react-app/app/components/search/partials/suggestions.jsx

```diff
@@ -23,11 +30,50 @@ const Suggestions = ({suggestions, closeAndNavigate}) => {
                         key={idx}
                         marginTop={0}
                         variant="menu-link"
+                        style={{justifyContent: 'flex-start', padding: '8px 12px'}}
                     >
-                        <Text
-                            fontWeight="400"
-                            dangerouslySetInnerHTML={{__html: suggestion.name}}
-                        />
+                        <Flex align="center">
+                            {/* Reserve space for image for all, but only render if present */}
+                            <Box
+                                width="40px"
+                                height="40px"
+                                marginRight="16px"
+                                borderRadius="full"
+                                background="transparent"
+                                display="flex"
+                                alignItems="center"
+                                justifyContent="center"
+                                overflow="hidden"
+                            >
+                                {(suggestion.type === 'product' ||
+                                    suggestion.type === 'category') &&
+                                    suggestion.image && (
```

Comment:
should this be guarding the parent Box element, too? Otherwise you may end up rendering an empty box, or is that intentional?

Reply:
Intended so so that if there's not an image everything is still aligned